IdentityCore/src/controllers/ios/MSIDBrokerInteractiveController.m

```diff
@@ -169,8 +169,17 @@ - (void)acquireTokenImpl:(nonnull MSIDRequestCompletionBlock)completionBlock
         return;
     }
 
+    NSError *appTokenError = nil;
+    NSString *applicationToken = [self.brokerKeyProvider getApplicationToken:self.interactiveParameters.clientId error:&appTokenError];
+    
+    if (appTokenError)
```

Comment:
nit: we should check for applicationToken here, not the error. Just a practice we did for all others

Reply:
Addressed.